config/environment.rb

```diff
@@ -3,3 +3,5 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+Capybara.ignore_hidden_elements = false
```

Comment:
We can look into removing this now, I believe it allows Capybara to see hidden CSS elements, but not 100% sure